### PR TITLE
Update description for verbosity

### PR DESCRIPTION
**Pull request by xinydev**
Fixes https://github.com/kubernetes-sigs/kind/issues/1624

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test
/lgtm
/approve
since matches Ben proposed solution in 
https://github.com/kubernetes-sigs/kind/issues/1624#issuecomment-868299105

File: pkg/cmd/kind/root.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		"verbosity",
 		"v",
 		0,
-		"info log verbosity",
+		"info log verbosity. higher value produces more output",
```
Comment (aojea) [inline pkg/cmd/kind/root.go:72]:
```suggestion
		"info log verbosity, higher value produces more output",
```
or we start with caps after the dot or we use a comma, right?

**Comment by aojea** [comment]
/hold

grammatical question :)

**Comment by aojea** [comment]
> /hold
> 
> grammatical question :)

let's wait for an english native
@BenTheElder

**Comment by BenTheElder** [comment]
comma sounds good
thank you!
/lgtm
/approve

**Comment by BenTheElder** [comment]
thanks!

**Pull request merged**